src/AppInstallerCommonCore/NetworkSettings.cpp

```diff
@@ -29,9 +29,13 @@ namespace AppInstaller::Settings
         // We only use WinINet if specified by settings, or if we want to use proxy (as DO does not support that)
         InstallerDownloader setting = User().Get<Setting::NetworkDownloader>();
 
-        if (setting != InstallerDownloader::WinInet && m_proxyUri)
-        {
-            AICLI_LOG(Core, Info, << "Forcing use of wininet for download as DO does not support proxy");
+        if (setting == InstallerDownloader::WinInet || m_proxyUri)
```

Comment:
This feels like it would be more clear (and future proof in the face of a new downloader being added) with something like:
```
if (m_proxyUri && setting != InstallerDownloader::WinInet)
{
    AICLI_LOG(Core, Info, << "Forcing use of wininet for download proxy support");
    setting = InstallerDownloader::WinInet;
}
else if (setting == InstallerDownloader::Default)
{
    setting = InstallerDownloader::DeliveryOptimization;
}

return setting;
```